relevanceai/utils/decorators/analytics.py

```diff
@@ -31,8 +31,6 @@ def wrapper(*args, **kwargs):
         if os.getenv(TRANSIT_ENV_VAR) == "TRUE":
```

Comment:
Add comment: 

```
# This env variable is used to track whether to send to MixPanel or not. If True, it does not and immediately
goes to function. Otherwise, it logs to MixPanel. 
```